examples/data-solutions/data-playground/main.tf

```diff
@@ -71,27 +79,73 @@ module "vpc-firewall" {
   project_id   = module.project.project_id
   network      = module.vpc.name
   admin_ranges = [var.vpc_config.ip_cidr_range]
+  custom_rules = {
+    #TODO Remove and rely on 'ssh' tag once issues/9273 is fixed
```

Comment:
Does this comment refer to an issue in the Fabric framework on Github? I don't see one with that number in the list https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/issues

Reply:
The google provider issue: https://github.com/hashicorp/terraform-provider-google/issues/9273